src/molecules/tooltips/tooltip.tsx

```diff
@@ -0,0 +1,94 @@
+import Tippy from "@tippyjs/react";
+import { followCursor } from "tippy.js";
+import React from "react";
+import { StringUtils } from "andculturecode-javascript-core";
+
+// -------------------------------------------------------------------------------------------------
+// #region Interfaces
+// -------------------------------------------------------------------------------------------------
+
+export interface TooltipProps {
+    /**
+     * Required. The element that triggers the tooltip on hovering.
+     * Surround the trigger with the `<Tooltip>` component.
+     */
+    children: React.ReactElement;
```

Comment:
Consider refactoring to use `PropsWithChildren` vs. manually defining children

Reply:
Required in this case being this component 'requires' children

Reply:
Oh, gotcha! 👍